src/olympia/devhub/views.py

```diff
@@ -1385,7 +1383,7 @@ def decorator(f):
         @functools.wraps(f)
         def wrapper(request, *args, **kw):
             step = outer_step
-            max_step = 6
+            max_step = 5
```

Comment:
Yeah, that's the problem.  Would need to replace SubmitStep with some other way of checking completeness.  It's a worthwhile task I suppose.